hw/ip/pwrmgr/data/pwrmgr.hjson

```diff
@@ -94,7 +94,13 @@
       fields: [
         { bits: "0",
           name: "EN",
-          desc: "Configuration enable.  Once set to 0, it can no longer be configured to 1",
+          desc: '''
+            Configuration enable.
+
```

Comment:
@weicaiyang 
can you have a look now? Does this feel more clear?

Reply:
Thanks for updating the description. It makes sense and is very clear now.